### ISSUE TITLE
CoreML/Metal?

### ISSUE THREAD

**Issue by alelordelo** [s1ddok/pix2pix-s4tf#1]
Awesome stuff!

Any plans for a CoreML/Metal model?

Cheers

**Comment by s1ddok** [comment]
There is a great article out there on how to convert S4TF models to CoreML. I'm more interested in training side right now, but converting should be more or less straightforward. I would love to see a contribution on this

**Comment by alelordelo** [comment]
Yup, converting is not a problem, you can easily  do that. What I am looking for is a Metal/CoreML implementation for training pix2pix, so you could train on a mac (like you do with CreateML for ex).

**Comment by s1ddok** [comment]
Oh, I see. Honestly I'm not sure if Metal or CoreML is capable of doing such a complex training. Even if it is, I think it is still in very early stage to invest time into such trainings, even though I would LOVE to use my Mac instead of remote linux server

**Comment by alelordelo** [comment]
Did you check this? 😉


https://developer.apple.com/wwdc20/10677 

Check out Build customized ML models with the Metal Performance Shaders Graph from #wwdc20

A GAN trained on iPad Pro

**Comment by s1ddok** [comment]
Yup, but it basically requires you to reimplement the whole thing from scratch, S4TF does not support it
